### ISSUE TITLE
Feature request: Documentation tf.keras.applications missing for TF 1.11

### ISSUE THREAD

**Issue by sibyjackgrove** [tensorflow/tensorflow#22597]
### Describe the problem
The documentation for tf.keras.applications is missing for version 1.11 and I am getting a page not found error (see below image). Documentation is available for 1.10. While both have the same documentation, 1.11 comes with `mobilenet_V2` which was not available in 1.10.

### Source code / logs
![image](https://user-images.githubusercontent.com/25213730/46215172-50ef2180-c302-11e8-9cd4-8e12ba62c1e1.png)

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
Mobile device

**Comment by av8ramit** [comment]
I can confirm. [Link](https://www.tensorflow.org/api_docs/python/tf/keras/applications) @MarkDaoust any ideas? @sibyjackgrove you can ignore the @tensorflowbutler request for more information.

**Comment by sibyjackgrove** [comment]
Thanks, for the update.

**Comment by harshini-gadige** [comment]
@MarkDaoust Hi Mark, any update on the fix ? Please share the PR # if it's created. Thanks !

**Comment by MarkDaoust** [comment]
They're partially fixed.

https://www.tensorflow.org/api_docs/python/tf/keras/applications
https://www.tensorflow.org/api_docs/python/tf/keras/preprocessing

Those main pages don't 404 anymore, but the contents could be better.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by mohantym** [comment]
Hi @sibyjackgrove ! The above links are working fine now in TF 2.6 . Closing this issue ,As it seem to be resolved  in latest versions ,Feel free to create a new issue if you face any problems further. Thank you!

**Comment by mohantym** [comment]
Are you satisfied with the resolution of your issue?
[Yes](https://goo.gl/forms/Oe0tEvODFRoI2gJF3)
[No](https://goo.gl/forms/fUjzOfrtkFbrOT8d2)
